core/src/Kokkos_View.hpp

```diff
@@ -1565,8 +1559,10 @@ class View : public ViewTraits<DataType, Properties...> {
         arg_N0, arg_N1, arg_N2, arg_N3, arg_N4, arg_N5, arg_N6, arg_N7));
   }
 
+  template <class PointerType, class = std::enable_if_t<std::is_convertible_v<
```

Comment:
Should we do `std::is_same` instead?

Reply:
I think it needs to be convertible to work with e.g. non const pointer for a const view

Reply:
Yes Christian already pointed to me it was a silly suggestion